src/v0/destinations/intercom/deleteUsers.js

```diff
@@ -25,20 +38,16 @@ const userDeletionHandler = async (userAttributes, config) => {
         }
       };
       const resp = await httpSend(requestOptions);
-      if (!resp || !resp.response) {
-        throw new CustomError("Could not get response", 500);
-      }
-      if (
-        resp &&
-        resp.response &&
-        resp.response.response &&
-        resp.response.response.status !== 200 &&
-        resp.response.response.status !== 404 // this will be returned if user is not found. Will send successful to server
```

Comment:
Previously, `404` is treated as a success.

Reply:
This is fixed